app/actions/TrezorActions.js

```diff
@@ -49,16 +48,6 @@ export const TRZ_TREZOR_ENABLED = "TRZ_TREZOR_ENABLED";
 // enableTrezor attepts to start a connection with connect if none exist and
 // connect to a trezor device.
 export const enableTrezor = () => (dispatch, getState) => {
-  const walletName = selectors.getWalletName(getState());
-
-  if (walletName) {
-    const config = wallet.getWalletCfg(
-      selectors.isTestNet(getState()),
-      walletName
-    );
-    config.set(cfgConstants.TREZOR, true);
-  }
-
```

Comment:
Alright to just set this when the wallet is created? If it trezor/ledger it will always be so. There's no reason to set it over and over.